src/emit/substrate/target.rs

```diff
@@ -1409,16 +1418,23 @@ impl<'a> TargetRuntime<'a> for SubstrateTarget {
                     )
                     .into()
             }
-            codegen::Expression::Builtin(_, _, codegen::Builtin::Gasleft, _) => {
+            codegen::Expression::Builtin {
+                builtin: codegen::Builtin::Gasleft,
+                ..
+            } => {
                 get_seal_value!("gas_left", "seal_gas_left", 64)
             }
-            codegen::Expression::Builtin(_, _, codegen::Builtin::Gasprice, expr) => {
+            codegen::Expression::Builtin {
+                builtin: codegen::Builtin::Gasprice,
+                args: values,
```

Comment:
```suggestion
                args,
```

Reply:
If you prefer `values` over `args`, we can name the field as so.

Reply:
I called this `values` first and then changed my mind. I've fixed it.